cmd/image-builder-db-test/main_test.go

```diff
@@ -364,77 +364,82 @@ func testBlueprints(t *testing.T) {
+	bp1v2Body, err := json.Marshal(bp1v2)
 	require.NoError(t, err)
 
-	newVersionId := uuid.New()
-	err = d.UpdateBlueprint(newVersionId, entry.Id, ORGID1, "new name", "new desc", bodyUpdated1)
+	bp1v2Id := uuid.New()
+	err = d.UpdateBlueprint(bp1v2Id, bp1Entry.Id, ORGID1, bp1v2.Name, bp1v2.Description, bp1v2Body)
 	require.NoError(t, err)
-	entryUpdated, err := d.GetBlueprint(entry.Id, ORGID1, ANR1)
+	bp1v2Entry, err := d.GetBlueprint(bp1Entry.Id, ORGID1, ANR1)
 	require.NoError(t, err)
-	bodyUpdated2, err := v1.BlueprintFromEntry(entryUpdated)
+	bp1v2FromEntry, err := v1.BlueprintFromEntry(bp1v2Entry)
 	require.NoError(t, err)
-	require.Equal(t, updated, bodyUpdated2)
-	require.Equal(t, updated.Version, entryUpdated.Version)
-	require.Equal(t, "new name", entryUpdated.Name)
-	require.Equal(t, "new desc", entryUpdated.Description)
+	require.Equal(t, bp1v2, bp1v2FromEntry)
+	require.Equal(t, bp1v2.Version, bp1v2Entry.Version)
+	require.Equal(t, bp1v2.Name, bp1v2Entry.Name)
+	require.Equal(t, bp1v2.Description, bp1v2Entry.Description)
 
-	require.NotEqual(t, b1, bodyUpdated2)
+	require.NotEqual(t, bp1, bp1v2FromEntry)
 
-	newBlueprintVersion := v1.BlueprintV1{
+	bp1v3 := v1.BlueprintV1{
 		Version:     3,
 		Name:        "name should not be changed",
 		Description: "desc should not be changed",
 	}
-	newBlueprintVersionBody, err := json.Marshal(newBlueprintVersion)
+	bp1v3Body, err := json.Marshal(bp1v3)
 	require.NoError(t, err)
-	newBlueprintId := uuid.New()
-	err = d.UpdateBlueprint(newBlueprintId, entry.Id, ORGID2, "name", "desc", newBlueprintVersionBody)
+	bp1v3Id := uuid.New()
+	err = d.UpdateBlueprint(bp1v3Id, bp1Entry.Id, ORGID2, bp1v3.Name, bp1v3.Description, bp1v3Body)
 	require.Error(t, err)
-	entryAfterInvalidUpdate, err := d.GetBlueprint(entry.Id, ORGID1, ANR1)
+	entryAfterInvalidUpdate, err := d.GetBlueprint(bp1Entry.Id, ORGID1, ANR1)
 	require.NoError(t, err)
-	bodyNotUpdated, err := v1.BlueprintFromEntry(entryAfterInvalidUpdate)
+	fromEntryAfterInvalidUpdate, err := v1.BlueprintFromEntry(entryAfterInvalidUpdate)
 	require.NoError(t, err)
-	require.Equal(t, updated, bodyNotUpdated)
-	require.Equal(t, updated.Version, bodyNotUpdated.Version)
-	require.Equal(t, "new name", bodyNotUpdated.Name)
-	require.Equal(t, "new desc", bodyNotUpdated.Description)
-	require.NotEqual(t, b1, bodyNotUpdated)
+	require.Equal(t, bp1v2, fromEntryAfterInvalidUpdate)
+	require.Equal(t, bp1v2.Version, entryAfterInvalidUpdate.Version)
+	require.Equal(t, bp1v2.Name, entryAfterInvalidUpdate.Name)
+	require.Equal(t, bp1v2.Description, entryAfterInvalidUpdate.Description)
 
-	newestBlueprintVersionId := uuid.New()
-	newestBlueprintId := uuid.New()
-	newestBlueprintName := "new name"
-	err = d.InsertBlueprint(newestBlueprintId, newestBlueprintVersionId, ORGID1, ANR1, newestBlueprintName, "desc", body)
+	bp2VersionId := uuid.New()
+	bp2Id := uuid.New()
+	bp2 := v1.BlueprintV1{
+		Version:     1,
+		Name:        "name",
+		Description: "desc",
+	}
```

Comment:
I'd suggest to add `bp2` prefix - i.e `bp2_name` To make the blueprint's name and description distinct and verbose (so tests won't be true negative).